nmdc_runtime/api/endpoints/queries.py

```diff
@@ -607,3 +607,73 @@ def _run_mdb_cmd(
         )
 
     return cmd_response
+
+
+def _run_delete_nonschema(
+    cmd: DeleteCommand, mdb: MongoDatabase = _mdb
+) -> DeleteCommandResponse:
+    """
+    Performs deletion operations similar to _run_mdb_cmd but skips the target_document
+    descriptor code that handles referential integrity checking.
```

Comment:
I changed "similar" (adjective) to "similarly" (adverb) to emphasize that the function performs the deletions in a similar way to the way `_run_mdb_cmd` does.

```suggestion
    Performs deletion operations similarly to _run_mdb_cmd but skips the target_document
    descriptor code that handles referential integrity checking.
```